__tests__/integration/atomic_swap.test.js

```diff
@@ -102,6 +103,7 @@ describe('partial tx proposal', () => {
     expect(tx.hash).toBeDefined();
 
     await waitForTxReceived(hWallet1, tx.hash);
+    await delay(1000); // This transaction seems to take longer than usual to complete
```

Comment:
This test is not related to the scope of this PR, but was failing constantly and was blocking the CI completion. This extra delay solved the issue.